source/docs/use/download/download.md

```diff
@@ -46,7 +43,7 @@ How you download HACS depends on your Home Assistant installation type. In the i
     3. Run the HACS download script.
 
         ```bash
-        wget -O - https://get.hacs.xyz | bash -
+        curl -Ls https://get.hacs.xyz | bash -
```

Comment:
Curl is available on fewer systems (debian does no longer ships it for instance), and as it does not really add any more to it, it should not be changed.
wget is also what is used inside that script, making this change add another requirement.